Essentials/src/com/earth2me/essentials/EssentialsPlayerListener.java

```diff
@@ -624,7 +624,12 @@ public void onPlayerInteract(final PlayerInteractEvent event) {
                     }
                 }
                 break;
-            case PHYSICAL:
+            case PHYSICAL:  			
+                if(event.getItem().getType().toString().contains("pressure_plate")) {
```

Comment:
This will never work for multiple reasons:
* `PlayerInteractEvent#getItem` returns the item used to interact with the block, not the block block that was interacted with - use `PlayerInteractEvent#getClickedBlock`.
* The string representation of Bukkit `Material`s is in uppercase.

Please also fix the spacing after `if`.